package.json

```diff
@@ -11,6 +11,17 @@
     "type": "git",
     "url": "git+https://github.com/flaviait/ng2-jspm-template.git"
   },
+  "scripts": {
+    "dist": "dist && npm run dist-bundle && cp src/index.html dist",
```

Comment:
_From practical test_:
The `main.css` does not get copied to the `dist` folder. As a result, there are no global styles in the dist version.